Production/govuk_ios/Builders/CoordinatorBuilder.swift

```diff
@@ -43,6 +43,7 @@ class CoordinatorBuilder {
         LaunchCoordinator(
             navigationController: navigationController,
             viewControllerBuilder: ViewControllerBuilder(),
+            appConfigService: container.appConfigService.resolve(),
```

Comment:
Have injected app config service into launch coordinator for now.
Existing logic is that app config is fetched on init of app config service so have left like that for now.